README.md

```diff
@@ -1,2 +1,75 @@
-# online-book-store
-Online Book store
+
+# Online Book Store Application
+
+## Introduction
+
+Welcome to the Online Book Store! This application serves as a digital marketplace where users can browse, search, purchase, and manage books online. Built to streamline the book acquisition process, it facilitates easy access to a wide range of books for reading enthusiasts.
+
+## Technologies and Tools
+
+This project uses a suite of advanced technologies and frameworks:
+- **Spring Boot**: Enables rapid development and deployment.
+- **Spring Security**: Ensures safe user interactions through comprehensive authentication and authorization.
+- **Spring Data JPA**: Simplifies CRUD operations and database management.
+- **Swagger (OpenAPI)**: Facilitates API documentation and testing.
+- **MySQL**: Provides a robust database management system.
+- **Docker**: Ensures consistent application execution across different environments.
+- **JWT (JSON Web Tokens)**: Secures information transmission.
+- **Liquibase**: Manages database schema migrations.
+
+## Functionalities
+
+- **AuthenticationController**: Manages user registration and login.
+- **BookController**: Facilitates book management including CRUD operations.
+- **CategoryController**: Organizes books into categories.
+- **OrderController**: Handles order placements and updates.
+- **ShoppingCartController**: Manages the shopping cart.
+
+![img.png](book-store-diagram.png)
+
+## Setup and Usage
```

Comment:
I would suggest to add 

`Make sure you have JDK and Docker installed`

After clone repo:
`Create .env file and fill in all necessary variables (You can find them in .env.sample)`. 
Of course you need to have .env.sample file

Also, add `Swagger is available for testing at 'url for swagger'`

Reply:
done